docs/markdown/Contributions/contributor-overview.md

```diff
@@ -0,0 +1,166 @@
+
+Bad titles:
+
+- Fix bug
+- Fix #8313
+- Add support for Thrift code generation by first adding the file `codegen.py`, then hooking it up, and finally adding tests
+
+Then, include a description. You can use the default template if you'd like, or use a normal description instead. Link to any corresponding GitHub issues.
+
+Finally—if you have the permissions—add exactly one of the following labels to your PR. Otherwise, a maintainer will do this for you:
+
+- `category:new feature` for new features
+- `category:user api change` for changes that affect how end-users interact with Pants
+- `category:plugin api change` for changes that affect how plugin authors interact with Pants internals
+- `category:performance` for changes focused on improving performance
+- `category:bugfix` for bugfixes
+- `category:documentation` for documentation changes, including logging and help messages
+- `category:internal` for miscellaneous, internal-facing changes
+
+Pick the first of these that applies to your change. I.e., if you have modified the user API in a change that also improves performance, use `category:user api change`.
+
+These labels are used to generate the changelist for each release.
+
+> 📘 Tip: Review your own PR
+> 
+> It is often helpful to other reviewers if you proactively review your own code. Specifically, add comments to parts where you want extra attention.
+> 
+> For example:
+> 
+> - "Do you know of a better way to do this? This felt clunky to write."
+> - "This was really tricky to figure out because there are so many edge cases. I'd appreciate extra attention here, please."
+> - "Note that I did not use a dataclass here because I do not want any of the methods like `**eq**` to be generated."
+
+> 📘 FYI: we squash merge
+> 
+> This means that the final commit message will come from your PR description, rather than your commit messages. 
+> 
+> Good commit messages are still very helpful for people reviewing your code; but, your PR description is what will show up in the changelog.
+
+### CI
+
+We use GitHub Actions for CI. Look at the "Checks" tab of your PR.
+
+> 📘 Flaky tests?
+> 
+> We unfortunately have some flaky tests. If CI fails and you believe it is not related to your change, please comment about the failure so that a maintainer may investigate and restart CI for you.
+> 
+> Alternatively, you can push an empty commit with `git commit --allow-empty` to force CI to restart. Although we encourage you to still point out the flake to us.
+
+### Review feedback
+
+One or more reviewers will leave feedback. If you are confused by any of the feedback, please do not be afraid to ask for clarification!
+
+If we do not respond within 24 business hours, please gently ping us by commenting "ping" on your pull request or messaging on Slack asking if someone could please take a look.
+
+Once one or more reviewers have approved—and CI goes green—a reviewer will merge your change.
+
+> 📘 When will your change be released?
+> 
```

Comment:
Oh, interesting. This looks like an alternative to the `block` syntax?

Reply:
Right, this is Readme's new preferred syntax for blocks -- markdown blockquote syntax, with a leading emoji to indicate which sort of block it is.